apps/studio/src/store/modules/TabModule.ts

```diff
@@ -78,6 +88,9 @@ export const TabModule: Module<State, RootState> = {
       context.commit('set', [])
       context.commit('setActive', null)
     },
+    async reopenLastClosedTab(context) {
+      context.commit('reopenLastClosedTab')
```

Comment:
Here it should probably get the last closed tab, then call context.dispatch('add', clonedTab)